src/components/Products/Products.tsx

```diff
@@ -0,0 +1,20 @@
+interface Category {
+    id: number;
+    name: string;
+    image: string;
+    creationAt: string;
+    updatedAt: string;
```

Comment:
```suggestion
    creationAt: Date;
    updatedAt: Date;
```
they are a `Date` type